packages/buckets/src/buckets.spec.ts

```diff
@@ -271,6 +273,107 @@ describe('Buckets...', function () {
       }
     })
   })
+
+  describe('multiple pushes', function () {
+    it('should throw if roots are out of sync/order', async function () {
```

Comment:
This is just an additional test on using root directly. We want to make sure we get an error when we expect to get one.